style.css

```diff
@@ -815,8 +813,8 @@ a .nav-title,
 .site-footer .widget-area ul li a,
 .site-footer .widget-area ul li a:visited {
 	border: 0;
-	-webkit-transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
-	transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
+	box-shadow: inset 0 0 0 rgba( 0, 0, 0, 0), 0 0px 0 rgba(0,0,0, 0);
```

Comment:
Remove space after first ( in rgba() and add space after each comma.

It should be
```
box-shadow: inset 0 0 0 rgba(0, 0, 0, 0), 0 0px 0 rgba(0, 0, 0, 0);
```